src/app/login/login.component.ts

```diff
@@ -1,12 +1,17 @@
 import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute, Router } from '@angular/router';
+import {AuthProvider} from 'ngx-auth-firebaseui';
```

Comment:
Looks like you haven't run the formatter. Please make sure you do. You can also run `ng lint --fix` and push.

Reply:
done